apps/src/templates/studioHomepages/MarketingAnnouncementBanner.jsx

```diff
@@ -104,8 +114,12 @@ const MarketingAnnouncementBanner = ({announcement, marginBottom}) => {
     >
       {/* ID is used for easier targeting in Optimizely */}
       <div id="special-announcement-action-block" ref={bannerRef}>
-        <SpecialAnnouncementActionBlock
-          announcement={announcement}
+        <TwoColumnActionBlock
```

Comment:
Yes I'll update the comment on line 10, good catch.

I needed to have control over what function gets called when the button is clicked, this was the most straight forward way to pass an onClick to the button. The difference between this and the former implementation is that this will only render 1 button whereas the previous implementation would render 2 buttons if the data for 2 buttons was provided. I'll verify with Eric that they only intend to render 1 button. (I have a feeling they may not have known it could render 2 buttons)

Reply:
Yeah, I just verified with Eric who had no idea that he could define a second button for a banner in `announcments.json`